lib/SlackBot.js

```diff
@@ -81,24 +81,26 @@ function Slackbot(configuration) {
                     // might not be in a db
                     if (err || !team) {
                         slack_botkit.log.error('Received slash command, but could not load team');
-                    } else {
-                        message.type = 'slash_command';
-                        // HEY THERE
-                        // Slash commands can actually just send back a response
-                        // and have it displayed privately. That means
-                        // the callback needs access to the res object
-                        // to send an optional response.
+                    }
+                    // for now just continue
+                    // } else {
+                    message.type = 'slash_command';
```

Comment:
i'll remove this file/mods from the final PR, unless things change at your end wrt
https://github.com/howdyai/botkit/pull/40